lib/user.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'bcrypt'
+
+class User
+  attr_reader :id, :first_name, :last_name, :email, :password
+
+  def initialize(id:, first_name:, last_name:, email:, password:)
+    @id = id
+    @first_name = first_name
+    @last_name = last_name
+    @email = email
+    @password = password
+  end
+
+  def self.create(first_name:, last_name:, email:, password:)
+    hashed_password = BCrypt::Password.create(password)
+    connection = PG.connect(dbname: 'bnb_test')
+    result = connection.exec("INSERT INTO users (firstname, lastname, email, passworddigest) VALUES('#{first_name}','#{last_name}','#{email}','#{hashed_password}') RETURNING id, firstname, lastname, email, passworddigest;")
```

Comment:
Should multi-word columns be underscore separated or not? We've been using underscores. Don't think it matters which we do as long as we're consistent.

Reply:
I thought the general convention for multi-word columns in sql tables was just to ram them all together :shrug: